development/scripts/clone-repos.sh

```diff
@@ -21,7 +20,5 @@ do
     cd "${repo}"/
     git remote add upstream git@github.com:opensecuritycontroller/"${repo}".git
     git remote set-url --push upstream no_push
-    cd ../"${repo}"/.git/hooks 
-    ln -s ../../hooks/pre-push .
-    cd ../../../
+    cd ../

```

Comment:
the issue is that this is preventing me from pushing to my forked master branch as well which should be allowed.